api.md

```diff
@@ -1874,6 +1862,44 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_continuous_aggregate_policy() :community_function> [](add_continuous_aggregate_policy)
+
+Create a policy that refresh a windows in a continuous aggregate.
+
+#### Required Arguments [](add_continuous_aggregate_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `continuous_aggregate` | (REGCLASS) The continuous aggregate to add the policy for. |
+| `start_interval` | (INTERVAL) Start of the window relative to current time to refresh. |
+| `end_interval` | (INTERVAL) End of the window relative to current time to refresh. |
+| `schedule_interval` | (INTERVAL) Interval between refresh executions. |
+
+#### Optional Arguments [](add_continuous_aggregate_policy-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `if_not_exists` | (BOOLEAN) Set to true to avoid throwing an error if the continuous aggregate_policy already exists. A notice is issued instead. Defaults to false. |
+
+#### Returns [](add_continuous_aggregate_policy-returns)
+
+|Column|Description|
+|---|---|
+|`job_id`| (INTEGER)  TimescaleDB background job id created to implement this policy|
+
+
+#### Sample Usage [](add_continuous_aggregate_policy-examples)
+
+To add a policy that refreshes the last month once an hour:
+```sql
+SELECT add_continuous_aggregate_policy('conditions_summary',
+	start_interval => INTERVAL '1 month',
+	end_interval => INTERVAL '1 hour',
```

Comment:
The setting of end_interval is not motivated. And we never describe why you'd want to have a refresh offset/lag like this.

Reply:
Not sure what you mean. All the parameters given here are mandatory and have to be given.

Reply:
Above you say `add a policy that refreshes the last month once an hour`, which isn't strictly true since you have an "offset" of 1 hour via the `end_interval` (which takes 1 hour off of the month). This begs the question: "why is there an offset of 1 hour?". We never explain in the API documentation or the example why you'd like to have this 1 hour offset, so it just comes out of nowhere. It is there to avoid write amplification, but no such context or explanation is given.

The other parameters are explained, self-evident, or somewhat motivated by the text and given context.

Reply:
It is not clear if you want:
* To rewrite the documentation and add a discussion about the write amplification. I have deliberately skipped this because 1) it requires a discussion about recommended values and what yo use, which is not obvious, and 2) it is not suitable material for the API reference documentation and is more suitable for the "Using" section.
* To remove `end_interval`, which is not possible because it is mandatory to the function.
* Set `end_interval` to NULL, which raises a discussion about the write amplification again, and the API documentation is not a suitable place to discuss such things.

Reply:
(Not sure exactly what @erimatnor is proposing.  Agree that longer discussion should be in a "Using" section.  Why don't we revisit after we have a draft of that section, to see how we might reference it?)

Reply:
@mfreed my suggested change was already adopted as part of another comment, which is no longer visible.